<?xml version="1.0"?>
<launch>
    <node pkg="ufld_ros" type="lane_det" name="lane_det" output="screen" launch-prefix="">
        <param name="model_name" value="/media/ros/A666B94D66B91F4D/ros/test_port/Ultra_Fast_Lane_Detection_TensorRT/UFLD_C++/build/lane_det.engine"/>
        <param name="batch_size" value="1"/>
        <remap from="/usb_cam2/image_raw" to="/usb_cam2/image_raw"/>
		<!--              /kitti/camera_color_left/image_raw                 -->
		<!--              /usb_cam/image_raw               -->
    </node>
</launch>
